test/LogicalImagesTests.cpp

```diff
@@ -1,6 +1,18 @@
 #include <gtest/gtest.h>
 #include "json.hpp"
 #include "LogicalImages.hpp"
+#include "JSONFileReader.hpp"
+
+class AlgorithmA : public Algorithm
```

Comment:
I propose an AlgorithmStub name. It should also be in a separate hpp file (no need to have cpp)

Reply:
Will move to AlgorthmStub.hpp and also create there AlgorithmStubB, which returns false from solve function.